osscluster.go

```diff
@@ -1316,6 +1317,9 @@ func (c *ClusterClient) processPipelineNodeConn(
 	if err := cn.WithWriter(c.context(ctx), c.opt.WriteTimeout, func(wr *proto.Writer) error {
 		return writeCmds(wr, cmds)
 	}); err != nil {
+		if isBadConn(err, false, node.Client.getAddr()) {
```

Comment:
Is this correct? isBadConn() returning true for context cancelled errors as well.